cookbooks/bach_krb5/recipes/gems.rb

```diff
@@ -20,14 +20,9 @@
   action :nothing
 end.run_action(:install)
 
-file "correct-permissions-for-rkerberos" do
-  path "/opt/chef/embedded/lib/ruby/gems/1.9.1/specifications/rkerberos-0.1.3.gemspec"
-  mode 0644
+execute "correct-gem-permissions" do
```

Comment:
This seems really in appropriate for `bach_krb5` as it affects the whole `/opt/chef/embedded/lib/ruby/gems` tree. Ideally we should follow previous patterns such as:
Either set [umask](https://github.com/bloomberg/chef-bach/blob/d715bfe4fd6e33b739724fba3802e3adbb0b3ac7/cookbooks/bcpc/recipes/mysql_connector.rb#L124)
Or known to work for gems, set only that gem's permission as [graphite_handler](https://github.com/bloomberg/chef-bach/blob/d715bfe4fd6e33b739724fba3802e3adbb0b3ac7/cookbooks/bcpc/recipes/graphite_handler.rb#L17-L24) does.